lib/fastlane/plugin/wpmreleasetoolkit/actions/common/create_new_milestone_action.rb

```diff
@@ -11,14 +11,27 @@ def self.run(params)
 
         github_helper = Fastlane::Helper::GithubHelper.new(github_token: params[:github_token])
         last_stone = github_helper.get_last_milestone(repository)
+
         UI.message("Last detected milestone: #{last_stone[:title]} due on #{last_stone[:due_on]}.")
+
         milestone_duedate = last_stone[:due_on]
         milestone_duration = params[:milestone_duration]
         newmilestone_duedate = (milestone_duedate.to_datetime.next_day(milestone_duration).to_time).utc
         newmilestone_number = Fastlane::Helper::Ios::VersionHelper.calc_next_release_version(last_stone[:title])
         number_of_days_from_code_freeze_to_release = params[:number_of_days_from_code_freeze_to_release]
+        # If there is a review process, we want to submit the binary 3 days before its release
+        # Using 3 days is mostly for historical reasons where we release the apps on Monday and submit them on Friday.
```

Comment:
What do you think of this rewording?

```suggestion
        # Because of the app stores review process, we submit the binary 3 days before the intended release date.
        # Using 3 days is mostly for historical reasons, for a long time, we've been submitting apps on Friday and releasing them on Monday.
```